src/main/java/io/github/vhoyon/vramework/objects/LinkParams.java

```diff
@@ -15,18 +17,24 @@ public LinkParams(Link link, Object... params){
 	}
 	
 	@Override
-	public LinkableCommand getInstance() throws Exception{
+	public LinkableCommand getInstance() throws IllegalStateException{
 		
-		@SuppressWarnings("rawtypes")
-		ArrayList<Class> classes = new ArrayList<>();
+		List<Class<?>> classes = new ArrayList<>();
 		
 		for(Object object : params){
```

Comment:
Might as well optimize this with a Stream.
My original guess, might not compile :
```java
Class<?>[] constructorArray = Arrays.stream(params).map(Object::getClass).toArray(Class<?>[]::new);
```
This code should replace this :
```java
List<Class<?>> classes = new ArrayList<>();

for(Object object : params){
    classes.add(object.getClass());
}

... (Class<?>[])classes.toArray(new Class<?>[0]) ...
```